interactive_mode.py

```diff
@@ -0,0 +1,120 @@
+import subprocess
+import cv2
+from simple_term_menu import TerminalMenu
+# pylint: disable-next=import-error
+from TTS.api import TTS
+import reddit_integration as ri
+import utils
+import tts
+
+BACKGROUND_VIDEO_PATH = './video-generation/public/video/backgroundVideo.mp4'
+OUTPUT_FILE_BUZZ_WORDS = 'reddit_story_crazy_reading_reddit_short_funny_sad_insane'
+YES_NO_OPTIONS = ['Yes', 'No', 'Yes (start render)', 'No (start render)']
+
+def get_posts(videos_to_generate=None):
+    if videos_to_generate is None:
+        videos_to_generate = []
+    log = utils.get_logger()
+    # Get a reddit post
+    try:
+        post = ri.get_post(allow_over_18=True)
+    except ImportError as error:
+        log.warning(error)
+        get_posts(videos_to_generate)
+
+    subreddit = post['subreddit']
+    title = post['title']
+    body = post['selftext']
+    is_nsfw = post['over_18']
+    nsfw_string = ''
```

Comment:
nsfw_string= 'NSFW' if is_nsfw else ''

Reply:
Yes, either it says NSFW or nothing in the preview.